### PR TITLE
Rtmp stream key

### PR DESCRIPTION
**Pull request by mntrspace**

### CODE REVIEW

File: docs/server-side/v2/how-to-guides/configure-webhooks/webhook.md [hunk 1]
```diff
@@ -1549,4 +1548,42 @@ This event is sent when RTMP ingestion has successfully ended.
...
+| type          | `string`             | Type of the event <br/><br/> Example: ingest.end.success                                                                                                                                               |
+| error_message | `string`             | Error message for failure of ingestion start. Possible messages are: `"stream key disabled"`, `"internal error"`, `"no hls destinations"`, `"remote session active"`, `"already connected"`,  `"beam start error"` |
+| room_id       | `string`             | 100ms assigned room id <br/><br/> Example: 5f9edc6ac238215aec2312df                                                                                                                                    |
+| template_id   | `string`             | Template ID of the room <br/><br/> Example: 66112497abcd52312556c4gg                                                                                                                                   |
+
+
+#### Sample `ingest.end.success` event
```
Comment (pratishbodhale) [inline docs/server-side/v2/how-to-guides/configure-webhooks/webhook.md:1571]: should be `ingest.start.failure`

File: docs/server-side/v2/api-reference/stream-key/disable-rtmp-stream-key.mdx [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+|:--------|:---------|:------------|
+| room_id | `string` | mandatory   |
+
+### Errors
+
+| Description                   | Code |
+|:------------------------------|:-----|
+| Key not found                 | 404  |
+| Room not found                | 404  |
+| Room is disabled (not active) | 403  |
```
Comment (rtpriya) [inline docs/server-side/v2/api-reference/stream-key/disable-rtmp-stream-key.mdx:37]: Disabled Room is not just non-active rooms right? basically they are locked and cant be moved to active state until enabled (unlocked) - have we mentioned it as non-active rooms in other places also? else we can describe them as locked rooms, or just disabled

File: docs/server-side/v2/api-reference/stream-key/create-rtmp-stream-key.mdx [hunk 3]
```diff
@@ -1,11 +1,11 @@
 ---
 title: Create RTMP stream key
-nav: 3.169
+nav: 3.72
 ---
 
 Use this request to generate an RTMP stream key and RTMP URL for any room, enabling HLS live streaming from an external RTMP broadcaster to the specified room.
```
Comment (rtpriya) [inline docs/server-side/v2/api-reference/stream-key/create-rtmp-stream-key.mdx:6]: Can we also add a note that stream key APIs dont work on disabled rooms and room need to be enavled (using the room APIs) before using any streamkey API?

File: docs/server-side/v2/changelog/release-notes.mdx [hunk 4]
```diff
@@ -7,9 +7,9 @@ This Changelog highlights notable changes to the 100ms server-side API, such as
 
 ## 2023-10-17
 - Added support for RTMP ingestion to live stream on 100ms from a RTMP broadcaster. Added documentation for:
-    - [Create RTMP stream key for a specific room](/server-side/v2/api-reference/live-streams/create-rtmp-stream-key)
-    - [Get RTMP stream key and URL for a specific room](/server-side/v2/api-reference/live-streams/get-rtmp-stream-key)
-    - [Disable RTMP stream key for a specific room](/server-side/v2/api-reference/live-streams/disable-rtmp-stream-key)
+    - [Create RTMP stream key for a specific room](/server-side/v2/api-reference/stream-key/create-rtmp-stream-key)
```
Comment (rtpriya) [inline docs/server-side/v2/changelog/release-notes.mdx:10]: include the new field in hlsDestinations object in policyAPI - we are surfacing this to configure the beam autostop Timeout on a destination level

**Review by rtpriya — APPROVED** [review]

File: docs/server-side/v2/api-reference/stream-key/disable-rtmp-stream-key.mdx (hunk 2 above)
Comment (mntrspace) [reply docs/server-side/v2/api-reference/stream-key/disable-rtmp-stream-key.mdx:37]: > Disabled Room is not just non-active rooms right? basically they are locked and cant be moved to active state until enabled (unlocked)

Correct, these are disabled rooms which need to be enabled.
Also, when you say non-active rooms, you mean the session is not running in them, right? We can start RTMP In for those rooms. I tested this.

File: docs/server-side/v2/api-reference/stream-key/create-rtmp-stream-key.mdx (hunk 3 above)
Comment (mntrspace) [reply docs/server-side/v2/api-reference/stream-key/create-rtmp-stream-key.mdx:6]: done

File: docs/server-side/v2/how-to-guides/configure-webhooks/webhook.md [hunk 5]
```diff
@@ -1549,4 +1548,42 @@ This event is sent when RTMP ingestion has successfully ended.
...
+
+```json
+{
+  "version": "2.0",
+  "id": "********-****-****-****-***********",
+  "account_id": "***********************",
+  "app_id": "***********************",
+  "timestamp": "2023-10-16T12:36:58Z",
+  "type": "ingest.end.success",
```
[the remark this answers is not in the document]
Comment (mntrspace) [reply docs/server-side/v2/how-to-guides/configure-webhooks/webhook.md:1580]: is it ingest.failure or ingest.start.failure?

File: docs/server-side/v2/changelog/release-notes.mdx (hunk 4 above)
Comment (mntrspace) [reply docs/server-side/v2/changelog/release-notes.mdx:10]: @vivekd-100ms will push a PR separately along with release notes for this.
Will include when i create the how-to doc.

**Pull request merged**